python/tank/platform/environment.py

```diff
@@ -471,36 +509,69 @@ def __find_location_for_bundle(self, yml_file, parent_yml_data, section_name, bu
         :param section_name:        The name of the section that contains the bundle
         :param bundle_name:         The name of the bundle to find
         :param bundle_tokens:       A list of tokens representing the path to the parent data block
+        :param absolute_location:   Whether to ensure that the file path and tokens returned
+                                    references where the given bundle's location descriptor is
+                                    defined in full. This should be used when determining where
+                                    the bundles descriptor is defined in the config, but is not
+                                    necessarily the correct approach when looking for higher-level
+                                    configuration, such as getting
```

Comment:
such as getting what? 😄

Reply:
description feels really abstract. a concrete example to illustrate would be great here i think.